plugins/techdocs/package.json

```diff
@@ -39,6 +39,9 @@
     "@backstage/integration": "^0.5.8",
     "@backstage/integration-react": "^0.1.5",
     "@backstage/plugin-catalog-react": "^0.4.0",
+    "@backstage/plugin-search": "^0.4.5",
+    "@backstage/search-common": "^0.1.2",
+    "@backstage/techdocs-common": "^0.7.1",
```

Comment:
:/ 
This has already leaked into the `search` package as well. I'll remove it from here but should add additional note that `todo` to clean up search also.

Reply:
I think the search-common stuff already out there is okay because the intention is for that package to contain both FE/BE code (and right now only contains types anyway).